src/test/java/teammates/test/cases/browsertests/InstructorFeedbackResultsPageResponseCommentUiTest.java

```diff
@@ -28,7 +28,7 @@ public void testFeedbackResponseCommentActions() throws IOException {
 
         ______TS("Failure case: add empty feedback response comment using comment modal in question view");
 
-        resultsPage = loginToInstructorFeedbackResultsPage("IFResultsResponseCommentUiT.instr", "Open Session");
```

Comment:
Shall we use what you used previously? Double R makes it easy to miss typos.

Reply:
The previous one was too long and caused it to give error. That is why I shortened it.